spec/models/admin_set_spec.rb

```diff
@@ -76,16 +76,64 @@
     expect(subject.reload.description).to eq ["description"]
   end
 
+  describe "#default_set?" do
+    context "with default AdminSet" do
+      before do
+        subject.id = described_class::DEFAULT_ID
+        subject.save!
+      end
+      it "returns true" do
+        expect(subject.default_set?).to be true
```

Comment:
Nice. I wasn't aware of these ;) I'll get them updated and try to remember to use them going forward.